scripts/st2bootstrap-el6.sh

```diff
@@ -252,8 +252,13 @@ configure_st2_user () {
   # Generate ssh keys on StackStorm box and copy over public key into remote box.
   # NOTE: If the file already exists and is non-empty, then assume the key does not need
   # to be generated again.
+  ELMAJVER=$(cat /etc/redhat-release | sed 's/[^0-9.]*\([0-9.]\).*/\1/')
   if ! sudo test -s ${SYSTEM_HOME}/.ssh/stanley_rsa; then
-    sudo ssh-keygen -f ${SYSTEM_HOME}/.ssh/stanley_rsa -P ""
+    if [[ "$ELMAJVER" == 8 ]]; then
+        # EL8 added -m PEM to force RSA PEM format
+        PEM="-m PEM"
```

Comment:
@armab - since there is an include file, I didn't have time/knowledge on how to upgrade the script `generate_file_installer_scripts.py` to only update the EL8 script. If necessary, I could build a new include or attack somehow, looking for suggestions. 

As to the reason why we need `-m PEM` on EL8, upon reviewing the [ssh-keygen manpage](http://man7.org/linux/man-pages/man1/ssh-keygen.1.html) 

`ssh-keygen will by default write keys in an OpenSSH-specific format.
     This format is preferred as it offers better protection for keys at
     rest as well as allowing storage of key comments within the private key
     file itself.  The key comment may be useful to help identify the key.
     The comment is initialized to “user@host” when the key is created, but
     can be changed using the -c option.

     It is still possible for ssh-keygen to write the previously-used PEM
     format private keys using the -m flag.  This may be used when generat‐
     ing new keys, and existing new-format keys may be converted using this
     option in conjunction with the -p (change passphrase) flag.`

Other ST2 install scripts all use older versions of `ssh-keygen` which default to `PEM`, so I added the flag to maintain consistency ( the change requires other changes to move to the new openssl key format ), which I assumed would be better left to a different PR.

Reply:
That makes a lot of sense now, thanks for explaining!

I think a quick reasoning behind this would be great to have as code comment.

Reply:
Also, in fact if other installers already relying on `PEM` by default, we can just add that flag globally for everything. This will remove EL8 conditional.